src/apps/courierTracking.tsx

```diff
@@ -0,0 +1,196 @@
+        </ColContainer>
+        <ColContainer key="toAddress">
+          <div className="heading">To</div>
+          <br />
+          <div>{_.upperFirst(info.to_city)}</div>
+          <br />
+          <em>{info.to_pincode}</em>
+        </ColContainer>
+      </RowContainer>
+      <br />
+      <RowContainer>
+        <p>
+          {" "}
+          <b>Current Status</b>: {data.status.current_status_body} <br />
+          <b>Location</b>: {data.status.current_status_location} <br />
+          <em>Updated on: {data.status.current_status_time} </em>
+        </p>
+      </RowContainer>
+      <PickrrLink href={trackingUrl} target="_blank">
+        View on Pickrr
+      </PickrrLink>
+    </InfoContainer>
+  );
+}
+
+const triggerWords = [
+  "track",
+  "pickrr",
+  "courier",
+  "tracking number",
+  "delivery status",
+  "current status",
+];
+const numberPattern = /\d+/g;
+
+/**
+ * The UI logic of the app.
+ */
+function Component(props: InstantAppProps) {
+  const data = props.data;
+  const statusArray = _.reverse(
+    _.flatten(
+      _.map(data.track_arr, (track_object) => track_object.status_array)
+    )
+  );
+  return (
+    <TrackingContainer>
+      <h1>Courier Tracking Status</h1>
+      {data.err && <div>{data.err}</div>}
+      <TrackingInfo data={data} />
+
+      <h2>Updates</h2>
+      <StatusArrayContainer>
+        {_.map(statusArray, (statusObject) => {
+          return (
+            <RowContainer>
+              <p>
+                <span className="statusHeading">
+                  {statusObject.status_body}
+                </span>{" "}
+                <br />
+                <small>at</small> {statusObject.status_location} <br />
+                <small>on</small> <em>{statusObject.status_time}</em>
+              </p>
+            </RowContainer>
+          );
+        })}
+      </StatusArrayContainer>
+    </TrackingContainer>
+  );
+}
+
+async function queryToData({
+  query,
+}: queryToDataInput): Promise<queryToDataOutput> {
+  // If the query does not contain the following words, do not trigger the app
+  // `define`, `meaning`
+  if (!isTriggered(query, triggerWords, { substringMatch: true })) return;
+  // Extract tracking number from the query
+  let trackingNumbers = query.match(numberPattern);
+  const baseUrl =
+    "https://cfapi.pickrr.com/plugins/tracking/?format=json&tracking_id=";
+  let results = await Promise.all(
+    _.map(trackingNumbers, (number) => {
+      return fetch(baseUrl + _.toString(number))
+        .then((response) => response.json())
+        .then((responseData) => {
+          let result = null;
+          if (responseData && !responseData.err) {
+            result = responseData;
+          }
+          return result;
+        });
+    })
+  );
+  results = _.compact(results);
+
+  return results[0];
+}
+
+const CourierTracking = {
+  name: "Courier Tracking",
+  description: "I am a template app, please change this description",
```

Comment:
```suggestion
  description: "View any courier package status using Pickrr API",
```

Reply:
Oops, missed this :P 
Thanks for the change :)